drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/test/java/org/drools/workbench/screens/scenariosimulation/backend/server/DMNScenarioValidationTest.java

```diff
@@ -205,6 +218,21 @@ private void createDMNType(String decisionName, String rootType, String... steps
         mapOfMockDecisions.put(decisionName, decisionNodeMock);
     }
 
+    private void createDMNTypeWithAllowedValues(String decisionName, String rootType, String step, List<DMNUnaryTest> allowedTypes) {
```

Comment:
Could we also add a test where  dmnType.getAllowedValues() == null || dmnType.getAllowedValues().isEmpty()? to check that    when(currentType.getBaseType()).thenReturn(baseType);
I mean to check SImple type and custom type.